src/FakeItEasy/ExceptionThrowerConfigurationExtensions.cs

```diff
@@ -130,6 +134,11 @@ public static IAfterCallSpecifiedConfiguration Throws<T1>(this IExceptionThrower
                 });
         }
 
+#if DISABLED
+
+        // This method already has a generic parameter, and it can't be inferred; if we add another, we make it unusable,
+        // especially since the new one is something unintelligible like IAnyCallConfigurationWithReturnTypeSpecificied<Foo>...
+

```

Comment:
>  3 classes implement `IOutAndRefParametersConfiguration.AssignsOutAndRefParametersLazily`

A quibble, but I only see two: `AnyCallConfiguration` and `RuleBuilder`.
Even if I'm right, that still leaves 10 methods.
The real reason I bring it up is that I'm doubting my counting. What have I done wrong?